api/tests/Api/ChecklistItems/UpdateChecklistItemTest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+namespace App\Tests\Api\ChecklistItems;
+
+use App\Tests\Api\ECampApiTestCase;
+
+/**
+ * @internal
+ */
+class UpdateChecklistItemTest extends ECampApiTestCase {
```

Comment:
Maybe 1 more test to check for loop (similar like UpdateRootColumnLayoutTest->testPatchColumnLayoutValidatesNoParentLoop)?

Reply:
done